MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -175,6 +175,13 @@ public GraphicsAdapter Adapter
         /// The metrics are reset every frame after draw within <see cref="GraphicsDevice.Present"/>. 
         /// </summary>
         public GraphicsMetrics Metrics { get { return _graphicsMetrics; } set { _graphicsMetrics = value; } }
+
+        private GraphicsDebug _graphicsDebug;
+
+        /// <summary>
+        /// Access debugging APIs for the graphics subsystem.
+        /// </summary>
+        public GraphicsDebug GraphicsDebug { get { return _graphicsDebug; } set { _graphicsDebug = value; } }
```

Comment:
Why is there a public setter here?  Who and why would someone replace this if it is created by the `GraphicsDevice` itself?